src/nvim/eval/typval.h

```diff
@@ -226,7 +226,7 @@ typedef TV_DICTITEM_STRUCT(1) ScopeDictDictItem;
 /// @warning Must be compatible with ScopeDictDictItem.
 ///
 /// Also used for a variable.
-typedef TV_DICTITEM_STRUCT() dictitem_T;
+typedef TV_DICTITEM_STRUCT(1) dictitem_T;
```

Comment:
cc @ZyX-I

Reply:
Based on what I know this variant will trigger array overrun warnings from various static analyzers. Needs to be limited to MSVC, probably by some other header which will define macros like `ARRAY_LEN_RUNTIME` (“array length determined at runtime”).

Reply:
@ZyX-I I changed it to use `...` and `__VA_ARGS__`.